astropy/units/imperial.py

```diff
@@ -78,6 +76,15 @@
          doc="nautical unit of speed: 1 nmi per hour")
 
 
+###########################################################################
+# FORCE
+
+def_unit('lbf', 4.4482216 * si.N, namespace=_ns,
```

Comment:
Should we define it in its official definition instead?  32.174049 * ft * lb * s**-2